website/docs/00-tutorial-issue-and-transfer-xudt.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Tutorial: Mint and Transfer xUDT From Scratch"
+sidebar_position: 1
+---
+
+# Tutorial: Mint and Transfer xUDT From Scratch
+
+Extensible User Defined Token ([xUDT](https://blog.cryptape.com/enhance-sudts-programmability-with-xudt)) is a programmable token standard on the CKB blockchain. This tutorial guides you through creating and transferring xUDT tokens from scratch.
+
+## Pre-requirement
+
+- Basic TypeScript syntax knowledge, including defining variables with type declaration
+- Basic CKB knowledge, including Cell and RPC
+
+### Initialize Project
+
+We'll use [Bun](https://bun.sh/), a lightweight runtime environment compatible with Node.js and TypeScript. It allows you to run the code directly without transpiling. Alternatively, you can use [ts-node](https://github.com/TypeStrong/ts-node) with Node.js
+
+```shell
+bun --version
+#> 1.1.12
+```
+
+### Create Project Directory and Initialize
+
+```sh
+mkdir xudt-from-scratch
+cd xudt-from-scratch
+bun init -y
+
+# based on the @ckb-lumos/lumos@0.0.0-canary-84521a5-20240530061434
+npm install @ckb-lumos/lumos@canary
+```
+
+This will create a basic project structure with necessary files.
+
+### Generate a Private Key
+
+We'll need a private key to act as the owner of the minted xUDT. Run the following command to generate a random private key:
+
+```shell
+bun -e "console.log('0x'+require('crypto').randomBytes(32).toString('hex'))"
```

Comment:
I would suggest using `openssl` to create the private key for 2 reasons

1. `openssl` is more general and mature
2. `bun` is not a widely used command, users should learn/enable it manually